packages/shared/src/components/PostOptionsMenu.tsx

```diff
@@ -365,12 +365,14 @@ export default function PostOptionsMenu({
         icon: (
           <MenuIcon
             Icon={
-              sourceSubscribe.haveNotifications ? BellSubscribedIcon : BellIcon
+              sourceSubscribe.haveNotificationsOn
+                ? BellSubscribedIcon
+                : BellIcon
             }
           />
         ),
         label: `${
-          sourceSubscribe.haveNotifications
+          sourceSubscribe.haveNotificationsOn
```

Comment:
since this is destructured above

```suggestion
         haveNotificationsOn
```